docs/introduction/partials/_getting-started-expo.md

```diff
@@ -2,24 +2,16 @@
 
 :::caution Note
 
-Expo support with Detox is entirely a community driven effort. We have no specific support in Detox for Expo applications (ejected or otherwise).
-
+Expo integration with Detox is entirely a community-driven effort.
+We have no special support for Expo projects, and we do not maintain any Expo-specific code or documentation.
 :::
 
-There are a few resources on the Internet covering how to set up Detox with Expo apps:
-
-- <https://github.com/fschoenfeldt/clean-expo-detox-testing>
-- <https://toshev.medium.com/setting-up-detox-with-a-expo-managed-react-native-project-89342dce4df>
-- <https://blog.expo.dev/testing-expo-apps-with-detox-and-react-native-testing-library-7fbdbb82ac87>
+There is a tutorial on how to set up Detox with Expo projects, which you can find on Expo official documentation site: [Running E2E tests on EAS Build](https://docs.expo.dev/build-reference/e2e-tests/). It is a great starting point for projects that are using Expo.
```

Comment:
There is a tutorial on [how to set up Detox with Expo projects][Expo guide], which you can find on the Expo official documentation site. It is a great starting point for projects that are using Expo.